src/Console/InstallCommand.php

```diff
@@ -83,10 +85,11 @@ protected function installTests()
                 (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/'.$stubStack.'/pest-tests/Feature', base_path('tests/Feature/Auth'));
             }
 
+            (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/'.$stubStack.'/pest-tests/Feature', base_path('tests/Feature'));
             (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/'.$stubStack.'/pest-tests/Unit', base_path('tests/Unit'));
             (new Filesystem)->copy(__DIR__.'/../../stubs/'.$stubStack.'/pest-tests/Pest.php', base_path('tests/Pest.php'));
         } elseif (! $spladeStack) {
-            (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/'.$stubStack.'/tests/Feature', base_path('tests/Feature/Auth'));
+            (new Filesystem)->copyDirectory(__DIR__.'/../../stubs/'.$stubStack.'/tests/Feature', base_path('tests/Feature'));
```

Comment:
@pascalbaljet Just to make you aware if necessary. I needed to do some manual work here while merging the upstream changes. Maybe some errors are co-related to this.

Reply:
Thanks! I'll also work on the new profile page feature and dark mode, though the Splade templates need an update for that as well.

Reply:
Do you want to open separate PR for that so that the changes are more clear? I will help reviewing the changes if you want.

Reply:
Ahh, the Laravel team also renamed the `App` stub folder to `app` 😅

I've migrated most of the profile stuff, but some tests still break in CI. I've also added many of the `dark:*` classes to prepare for a Splade release that supports dark mode. For now, it deletes the classes.

Reply:
I think updating some dependencies for PHP 8.2 would be enough. I will try to get it to work.